mentat/commands.py

```diff
@@ -279,9 +277,12 @@ async def apply(self, *args: str) -> None:
             return
 
         for i, (feature, score) in enumerate(results, start=1):
-            label = feature.ref()
-            if label.startswith(str(git_root)):
-                label = label[len(str(git_root)) + 1 :]
+            if feature.path.is_relative_to(session_context.cwd):
+                label = f"{feature.path.relative_to(session_context.cwd)}"
```

Comment:
Could feature.ref have an optional 'cwd' arg? It's quite useful elsewhere for quickly grabbing/comparing the path/interval.

Reply:
What would the `cwd` arg do in this case?

Reply:
```
    def ref(self, cwd: Optional[Path] = None):
        path_string = str(self.path) 
        if cwd and self.path.is_relative_to(cwd):
            path_string = self.path.relative_to(cwd)
        if self.level == CodeMessageLevel.INTERVAL:
            interval_string = f"{self.interval.start}-{self.interval.end}"
            return f"{path_string}:{interval_string}"
        return path_string

```